src/rvs_blas.cpp

```diff
@@ -60,26 +77,19 @@ rvs_blas::rvs_blas(int _gpu_device_index, int _m, int _n, int _k, int transA, in
     }
 
     if(ops_type == "hgemm") {
-        auto    A_row = transA == rocblas_operation_none ? m : k;
+        //auto    A_row = transA == rocblas_operation_none ? m : k;
         auto    A_col = transA == rocblas_operation_none ? k : m;
-        auto    B_row = transB == rocblas_operation_none ? k : n;
+        //auto    B_row = transB == rocblas_operation_none ? k : n;
         auto    B_col = transB == rocblas_operation_none ? n : k;
 
-        size_a = size_t(lda) * size_t(A_col);
-        size_b = size_t(ldb) * size_t(B_col);
-        size_c = size_t(ldc) * size_t(n);
-        size_d = size_t(ldc) * size_t(n);
+        size_a = size_t(lda) * A_col;
```

Comment:
Any recent change in rocblas on how matrix size is calculated for hgemm ?

Reply:
No change is hgemm size formula.  These changes are don't affect sizes unless the product of columns and leading dimension exceeded the maximum value of an int32.   This PR should allow these larger sizes to run correctly instead of overflowing the int32.